web/app/charts/MapChart.js

```diff
@@ -169,11 +173,29 @@ bluewave.charts.MapChart = function(parent, config) {
 
                     }else if(chartConfig.mapType === "Area"){
                         data.forEach(function(d){
-                            var state = d.state;
-                            for(var i = 0; i < states.features.length; i++){
-                                if(state == states.features[i].properties.code){
-                                    states.features[i].properties.inData = true;
-                                    states.features[i].properties.mapValue = d[chartConfig.mapValue];
+                            if(useCensusData){
+                                var location = d.premier_facility_location;
```

Comment:
Can we make this a little more generic? Maybe ask the user for a field with census region/division and try to parse it?

Reply:
Yea we can do that.

Reply:
This has been resolved in a recent commit.